packages/koenig-lexical/src/components/ui/cards/SignupCard.jsx

```diff
@@ -1,5 +1,296 @@
+        return (
+            <FileUploading progress={progress} />
+        );
+    }
+    return (
+        <>
+            <form onChange={onFileChange}>
+                <input
+                    ref={fileInputRef}
+                    accept='image/*'
+                    hidden={true}
+                    name="image-input"
+                    type='file'
+                />
+            </form>
+            {
+                backgroundImagePreview && (
+                    <div className="w-full">
+                        <div className="relative">
+                            <div className="flex w-full items-center justify-center">
+                                {
+                                    backgroundImageSrc ?
+                                        <>
+                                            <div className="group relative mb-4 w-full rounded">
+                                                <div className="absolute inset-0 rounded bg-gradient-to-t from-black/0 via-black/5 to-black/30 opacity-0 transition-all group-hover:opacity-100">
+                                                </div>
+                                                <div className="absolute top-5 right-5 flex opacity-0 transition-all group-hover:opacity-100">
+                                                    <button className="pointer-events-auto flex h-8 w-9 cursor-pointer items-center justify-center rounded bg-white/90 transition-all hover:bg-white" type="button" onClick={handleClearBackgroundImage}>
+                                                        <TrashIcon className="fill-grey-900 h-5 w-5 stroke-[3px] transition-all ease-linear group-hover:scale-105" />
+                                                    </button>
+                                                </div>
+                                                <img alt='backgroundHeaderImage' className="max-h-64 w-full rounded object-cover" data-testid="image-picker-background" src={backgroundImageSrc} />
+                                            </div>
+                                        </>
+                                        :
+                                        <button className="border-grey-300 bg-grey-50 dark:border-grey-800 dark:bg-grey-900 group flex h-[120px] w-full cursor-pointer flex-col items-center justify-center rounded-sm border border-dashed" type="button" onClick={openFilePicker}>
+                                            <FileUploadIcon className="fill-grey-700 h-5 w-5 stroke-[3px] transition-all ease-linear group-hover:scale-105" />
+                                            <span className="text-grey-700 px-1 text-[1.35rem] font-medium">Click to upload background image</span>
+                                        </button>
+                                }
+                            </div>
+                        </div>
+                    </div>
+                )
+            }
+
+        </>
+    );
+}
```

Comment:
We could refactor the `ImagePicker` into a separate component and use it for both Header + Signup cards?